netlify.toml

```diff
@@ -143,4 +143,9 @@
   from = "/home/*"
   to = "/"
   force = true
+
+[[redirects]]
+  from = "/get-started"
+  to = "/get-started/self-hosted"
+  force = true
```

Comment:
Can we add a redirect for `/get-started/cloud` too? I notice this page is still accessible. Interestingly, the back `Deployment Options` button direct a user to `/get-started` from this page & it doesn't redirect to `/get-started/self-hosted` 🤔  A redirect for `/get-started/cloud` should cover this though
![image](https://user-images.githubusercontent.com/59381432/175982426-ba4528c5-7abf-4208-b3d5-5079e1c412ac.png)

Reply:
This is a great catch, Becca. I have added the redirect here: a89bb93. 

I also posed the question of removing the Cloud page to Andy, Alex, and Fabiana and will return with a response. As you noted, it makes sense to me that we would remove the page entirely if "Sourcegraph Cloud" is being phased out of About.

Reply:
Per this [comment](https://sourcegraph.slack.com/archives/C03M3338LP5/p1656350207350549?thread_ts=1656349394.483309&cid=C03M3338LP5), it looks like we will retain "Get Started / Cloud", and Alex is redesigning the user flow. Let's keep this page so that it can be reused.